core/components/atoms/message/__stories__/variants/Appearance.story.jsx

```diff
@@ -4,16 +4,12 @@ import { Message, Text } from '@/index';
 // CSF format story
 export const appearanceWithoutTitle = () => {
   const appearances = ['alert', 'info', 'success', 'warning'];
-  const innerStyle = {
-    display: 'flex',
-    'align-items': 'center',
-    'flex-direction': 'column',
-  };
+
   return (
     <div className="Row">
       {appearances.map((appear, ind) => {
         return (
-          <div key={ind} style={innerStyle} className="mr-7 mb-7 w-25 d-flex">
+          <div key={ind} className="mr-7 mb-7 w-25 d-flex flex-column align-items-center">
```

Comment:
Please refactor the example to use `Row` and `Column` like https://codesandbox.io/s/trusting-https-ny9i1d?file=/index.js